server/feeders/reservationsgenerator.js

```diff
@@ -56,29 +55,38 @@ var feedOrionReservations = function() {
   };
 
   Object.keys(restaurantsData).forEach(function(element, pos) {
-    // Call orion to append the entity
-    var restaurantName = restaurantsData[pos].id + '-' + shortid.generate();
+
+    var reservationId = restaurantsData[pos].id + '-' + shortid.generate();
 
     var reservations = ['Cancelled', 'Confirmed', 'Hold', 'Pending'];
 
     var attr = {
       'type': 'FoodEstablishmentReservation',
-      'id': restaurantName,
-      'reservationStatus': utils.randomElement(reservations),
-      'underName': {},
-      'reservationFor': {},
-      'startTime': utils.getRandomDate().getTime(),
-      'partySize': utils.randomIntInc(1, 20)
+      'id': utils.asciiEncode(reservationId),
+      'reservationStatus': {
+        'value': utils.randomElement(reservations)
+      },
+      'underName': {
+        'type': 'Person',
+        'value': 'user' + utils.randomIntInc(1, 10)
+      },
+      'reservationFor': {
+        'type': 'FoodEstablishment',
+        'value': utils.fixedEncodeURIComponent(restaurantsData[pos].name)
```

Comment:
shouldn't this be the restaurant id?

I think we are encoding because Orion does not accept certain chars as value, right?